changelog/issue-1773.md

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
The format is incorrect, that's why the `changelog` is failing. Please recheck.

Reply:
Okay , but after removing the first line , it is still failing the yarn  changelog check

Reply:
@vishakhanihore